src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsExecution.java

```diff
@@ -39,6 +39,9 @@ public boolean start() throws Exception {
         System.out.println(templates);
 
         Node node = computer.getNode();
+        if (node == null) {
+            throw new Exception("Computer does not correspond to a live node");
```

Comment:
What can the user do to remedy this situation if it occurs? Or what causes this error? I am just thinking, how can the plugin help the user out of this situation if it occurs? I see some possible answers here: http://javadoc.jenkins-ci.org/hudson/model/Computer.html